extensions/ql-vscode/src/remote-queries/export-results.ts

```diff
@@ -100,29 +102,28 @@ async function exportResultsToGist(
 
 /**
  * Converts the results of a remote query to markdown and saves the files locally
- * in the query directory (i.e. the workspace directory where query results and logs are saved).
+ * in the query directory (where query results and metadata are also saved).
  */
 async function exportResultsToLocalMarkdown(
-  queryHistoryManager: QueryHistoryManager,
-  queryHistoryItem: QueryHistoryInfo,
+  queryDirectoryPath: string,
   query: RemoteQuery,
   analysesResults: AnalysisResults[]
 ) {
   const markdownFiles = generateMarkdown(query, analysesResults, 'local');
-  const basePath = await queryHistoryManager.getQueryHistoryItemDirectory(
-    queryHistoryItem
-  );
+  const exportedResultsPath = path.join(queryDirectoryPath, 'exported-results');
+  await fs.ensureDir(exportedResultsPath);
   for (const markdownFile of markdownFiles) {
-    const filePath = path.join(basePath, `${markdownFile.fileName}.md`);
+    const filePath = path.join(exportedResultsPath, `${markdownFile.fileName}.md`);
```

Comment:
Nice...so now all markdown files will live in a single directory. This will make it easier to manage and delete across exports.